joint_trajectory_controller/CMakeLists.txt

```
@@ -27,16 +27,16 @@ add_library(joint_trajectory_controller SHARED
 )
 target_include_directories(joint_trajectory_controller PRIVATE include)
 ament_target_dependencies(joint_trajectory_controller
-  "builtin_interfaces"
-  "controller_interface"
-  "control_msgs"
-  "hardware_interface"
-  "pluginlib"
-  "rclcpp"
-  "rclcpp_lifecycle"
-  "rcutils"
-  "realtime_tools"
-  "trajectory_msgs"
+  builtin_interfaces
```

```suggestion
  angles
  builtin_interfaces
```

You most likely need to set the dependency here.

---

and to that end I guess in the package.xml as well

---

added angles dependencies
8a5ac43